src/datascience-ui/common/constants.ts

```diff
@@ -584,10 +585,8 @@ export namespace CodeSnippets {
     ];
     export const ChangeDirectoryCommentIdentifier = '# ms-toolsai.jupyter added'; // Not translated so can compare.
     export const ImportIPython = '{0}\nfrom IPython import get_ipython\n\n{1}';
-    export const MatplotLibInitSvg = `import matplotlib\n%matplotlib inline\n${Identifiers.MatplotLibDefaultParams} = dict(matplotlib.rcParams)\n%config InlineBackend.figure_formats = ['svg', 'png']`;
-    export const MatplotLibInitPng = `import matplotlib\n%matplotlib inline\n${Identifiers.MatplotLibDefaultParams} = dict(matplotlib.rcParams)\n%config InlineBackend.figure_formats = ['png']`;
-    export const ConfigSvg = `%config InlineBackend.figure_formats = ['svg', 'png']`;
-    export const ConfigPng = `%config InlineBackend.figure_formats = ['png']`;
+    export const MatplotLibInit = `import matplotlib\n%matplotlib inline\n${Identifiers.MatplotLibDefaultParams} = dict(matplotlib.rcParams)\n`;
+    export const AppendSVGFigureFormat = `import matplotlib_inline.backend_inline\n${Identifiers.MatplotLibFigureFormats} = matplotlib_inline.backend_inline.InlineBackend.instance().figure_formats\n${Identifiers.MatplotLibFigureFormats}.add('svg')\nmatplotlib_inline.backend_inline.set_matplotlib_formats(*${Identifiers.MatplotLibFigureFormats})`;
```

Comment:
Found some interesting stuff looking this up. Using %config to try to fetch the backend formats actually doesn't work until you fully initialize %matplotlib inline, which I didn't want to do for this. Also the figure_format specification has changed a few times and can be a bit fussy between python versions about [] or {} syntax as it was moved from a list to a set at one point.

So instead moved to using the backend_inline classes directly.

Reply:
Handy working in the open source world since I could find the impls here https://github.com/ipython/matplotlib-inline/blob/541b26da2f49d0403994b91cb062e8c438920c44/matplotlib_inline/backend_inline.py